### DISCUSSION TITLE
Japanese Input is broken on editing a list item in description

### DISCUSSION THREAD

**Discussion by pachirel** [community/community#18630] [Projects and Issues] [answered]
Hi, I found an issue with Japanese input.

Editing list item doesn't work well. Please check this video.
Hyphen is disappeared on hitting return key when I input text in Japanese by IME.

I use Google Chrome (102.0.5005.61).

Similar with https://github.com/github-community/community/discussions/6358.

https://user-images.githubusercontent.com/440515/173726462-3c22658b-c24c-4b10-abbe-737b5a18abc9.mp4

Thanks!

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by pachirel** [reply]
I see. I revisited this issue, the behavior is improved. But, it's still a little bit weird. I'll report it to the bug report. Thanks!

**Comment by j0siepy** [comment, marked as answer]
Marking this as answered as it's been reported as a bug 🐛
